app/views/layouts/application.html.erb

```diff
@@ -60,6 +60,12 @@
               </a>
             <% end %>
 
+            <%- if request.path_info == '/news' %>
```

Comment:
With the `active?(path)` helper I introduced in ApplicationHelper, this could become a single line call: 

```ruby
<%= link_to "News", news_url, class: "header__nav-link #{active?(news_path)}" %>
```